stable/prometheus-operator/templates/prometheus-operator/serviceaccount.yaml

```diff
@@ -6,6 +6,10 @@ metadata:
   namespace: {{ template "prometheus-operator.namespace" . }}
   labels:
     app: {{ template "prometheus-operator.name" . }}-operator
+{{- if .Values.prometheusOperator.serviceAccount.annotations }}s
```

Comment:
the s at the end should probably be removed?

Reply:
Fixed values.yaml to make consistent.